content/en/community/demo/_index.md

```diff
@@ -1,12 +1,8 @@
 ---
 title: OpenTelemetry Demo
 linkTitle: Demo
+description:
+  The OpenTelemetry Demo is a microservice-based distributed system intended to illustrate the implementation of OpenTelemetry in a near real-world environment.
```

Comment:
This same sentence is needed in 3 places. I don't know the docs structure, but I'm thinking if there's a way to have it only in one place and include it from there. It would be easier when changes are needed.

Reply:
I'm not aware of how to template these descriptions. I'm happy to also move it and bring it down into the text itself, but this also removes the description from google search results.

Reply:
We can follow up on this, @chalin might have a suggestion/solution for this?

Reply:
Hi. The fact that this duplication exists is a bad smell IMHO. For a suggested alternative way of achieving this, see:

- #2158